<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteEditor</class>
 <widget class="QDialog" name="PaletteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QToolButton" name="bank6">
     <property name="text">
      <string>6</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="bank5">
     <property name="text">
      <string>5</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="21">
    <layout class="QHBoxLayout" name="widgetLayout"/>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="bank4">
     <property name="text">
      <string>4</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="bank7">
     <property name="text">
      <string>7</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QToolButton" name="bank8">
     <property name="text">
      <string>8</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="bank3">
     <property name="text">
      <string>3</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="bank2">
     <property name="text">
      <string>2</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="bank1">
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="14" colspan="4">
    <widget class="QPushButton" name="exportPal">
     <property name="text">
      <string>Export Palette</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10" colspan="4">
    <widget class="QPushButton" name="importPal">
     <property name="text">
      <string>Import Palette</string>
     </property>
    </widget>
   </item>
   <item row="1" column="18">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
